doc/dart_documentation_comment_specification.md

```diff
@@ -0,0 +1,478 @@
+
+## **6\. Associating Documentation with Elements**
+
+While the placement rules in [Section 3](#3.-placement-of-documentation-comments) define where a doc comment must be written, this section defines how documentation tools should associate a specific documentation block with a given code element, especially in cases where the association isn't direct.
+
+### **6.1. The General Rule**
+
+The documentation for a given Dart element is, by default, the single doc comment that *immediately precedes* its declaration in the source code.
+
+### **6.2. Inheritance and Special Association Rules**
+
+In some cases, the effective documentation for an element is determined by context, inheritance, or combination rules rather than by a direct preceding comment.
+
+#### **6.2.1. Overridden Members**
+
+When a member overrides an ancestor, its documentation is determined by a set of inheritance and precedence rules.
+
+* **Explicit Documentation:** If an overriding member does have its own doc comment, that comment takes full precedence. The documentation from any base members is ignored.
+
+* **Unambiguous Documentation Inheritance** If a member overrides a member from an ancestor and does not have its own doc comment, it inherits the documentation from the base member. This rule applies when the source of the documentation is unambiguous (e.g., overriding a single concrete method).  Documentation tools should display this inherited comment, ideally noting its origin (e.g., "Copied from BaseClass").
+
+* **Ambiguous Documentation from Multiple Ancestors** The behavior for resolving documentation when a member inherits conflicting documentation from multiple ancestors is *reserved for future standardization*. Currently, tools may handle this case at their discretion, or ignore inherited documentation in ambiguous scenarios. It is recommended that tools *issue a warning* when this scenario is encountered, prompting the user to provide explicit documentation to resolve the ambiguity.
+
+#### **6.2.2. Getters and Setters**
+
+A getter and a setter that share the same name are treated as a single *conceptual property* rather than the individual getter or setter.
+
+Documentation tools should handle this property with the following rules:
+
+* **Display:** The getter and setter should be presented as a single item in the final documentation.
+* **Precedence:**
+  * Documentation for the property should only be placed on the getter.
+  * The tooling should *issue a warning* if both a getter and its corresponding setter have unique doc comments.
+  * The only situation the documentation of the setter is considered is in the case of no getter existing.
+* **Reference:** A reference to a property name (e.g., `[value]`) resolves to the conceptual property rather than the individual getter or setter. This applies in all cases where a getter and setter share the same name, including:
+  * When an explicit getter (get value) and setter (set value(...)) are declared.
```

Comment:
I think the code samples here should be in backticks.

Reply:
done